### ISSUE TITLE
Ignore some rclone failure codes

### ISSUE THREAD

**Issue by dantebarba** [pfidr34/docker-rclone#40]
Hey

In some cases rclone likes to exit with a specific error code even when it's just a desirable execution result. Some flags like --max-transfer and --max-duration are very useful for rate limiting but rclone interprets them as an error

Could it be possible to add a "SUCCESS_CODES" flag. This flag if set with a list of codes will trigger an ignore on the /fail notification. By default the value would be just "0".

So instead of having

```bash
  else
    if [ "$RETURN_CODE" == 0 ]
    then
....
```
```bash
else
  IS_SUCCESS=$(echo "$SUCCESS_CODES" | grep -oP " $RETURN_CODE ")
  if [ ! -z "$IS_SUCCESS" ]
  then
```
Just FYI I'm complaining about this on rclone also because --max-duration shouldn't be returning error code as per documentation.

https://forum.rclone.org/t/max-duration-option-is-triggering-exit-with-error/39917

**Comment by pfidr34** [comment]
Hey @dantebarba, thanks for the issue log. 

I saw you just forked. If you end up coding this in your repo, I would be happy to take this as a pull request if you submit it.

**Comment by dantebarba** [comment]
Looks like they are going to release the fix in 1.64. I won't wait on that. I'll start doing some tests and then if everything goes ok I'll create the PR.

**Comment by pfidr34** [comment]
Merged and pushed to DockerHub!
